common/script_values/00_age_values.txt

```diff
@@ -1,7 +1,7 @@
 ﻿
 #Childhood
 childhood_personality_age = 3
-childhood_education_start_age = 6
+childhood_education_start_age = 6 #Can't change these, doesn't matter since they're under 18
 
 adulthood_start_age = 16
```

Comment:
Why you can't?

Reply:
> 
> 
> Why you can't?

Lot's of errors relating to the value. Can't quite remember what they were.

Reply:
> > Why you can't?
> 
> Lot's of errors relating to the value. Can't quite remember what they were.

Maybe it was because you didn't put in brackets and then in value?

Reply:
> 
> 
> > > Why you can't?
> > 
> > 
> > Lot's of errors relating to the value. Can't quite remember what they were.
> 
> Maybe it was because you didn't put in brackets and then in value?

Will try it

Reply:
Worked